src/template/stats/projects-popup.html

```diff
@@ -0,0 +1,24 @@
+<div class="projects-popup">
```

Comment:
there are unnecessary two-way bindings in this file. Better replace them with one one time bindings